pkgs/data/fonts/font-awesome/default.nix

```diff
@@ -21,13 +21,14 @@ let
         Font Awesome gives you scalable vector icons that can instantly be customized.
         This package includes only the OTF font. For full CSS etc. see the project website.
       '';
-      homepage = "http://fortawesome.github.io/Font-Awesome/";
+      homepage = "https://fontawesome.com/";
       license = licenses.ofl;
       platforms = platforms.all;
       maintainers = with maintainers; [ abaldeau johnazoidberg ];
     };
   };
-in {
+in
+{
   # Keeping version 4 because version 5 is incompatible for some icons. That
```

Comment:
Not sure, shall we update this comment to also mention version 6?

Reply:
I think so, what about `# Keeping version 4 and 5 because version 6 is incompatible for some icons.`?

Reply:
Sounds good to me